intel_extension_for_transformers/qbits/dispatcher/src/bestla_weightonly_dispatcher.cpp

```diff
@@ -331,7 +331,7 @@ void parse_gemm_core_online(woq_config_param* p, woq_runtime_ctx* ctx) {
     }
     TORCH_CHECK(false, "Qbits: device ISA must support BTLA_ISA::AVX2 when compute_type==fp32");
   }
-  if (p->compute_type == "bf16") {
+  if (p->compute_type == "bf16" && p->blocksize % 32 == 0) {
```

Comment:
only when the compute dtype is bf16 , we cannot support block size %32 != 0 ?

Reply:
int8 compute-dtype also has some blocksize constraints, but we already added related checks before.
this constraint comes from BesTLA.